### PR TITLE
doc: Updates the Public -> Private preview in docs for resource-policy

### PR DESCRIPTION
**Pull request by EspenAlbert**
Updates the Public -> Private preview in docs for resource-policy

Link to any related issue(s): CLOUDP-277628

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code
- [ ] I have tested the CDK constructor in a CFN stack. See [TESTING.md](../TESTING.md)
- [ ] If changes include removal or addition of 3rd party GitHub actions, I updated our internal document. Reach out to the APIx Integration slack channel to get access to the internal document.

### CODE REVIEW

File: src/l1-resources/resource-policy/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 The official [MongoDB Atlas](https://www.mongodb.com/) AWS CDK resource for Node.js.
 
 > **NOTE**: 
-> - **Atlas Resource Policies functionality is currently in Preview, you'll need to enable `Atlas Resource Policies` Beta Feature in your organization to use it.** 
+> - **Resource Policies is currently in Private Preview, if you want to test it, contact [MongoDB Support](https://www.mongodb.com/services/support) to enable `Atlas Resource Policies` Beta Feature in your organization.**
```
Comment (marcosuma) [inline src/l1-resources/resource-policy/README.md:6]: not sure about the `test` word here

Comment (EspenAlbert) [reply src/l1-resources/resource-policy/README.md:6]: maybe `try` is better?

Comment (marcosuma) [reply src/l1-resources/resource-policy/README.md:6]: yes, but I'd wait for docs team to confirm :)

Comment (kyuan-mongodb) [reply src/l1-resources/resource-policy/README.md:6]: I agree that `try` sounds better here.

Comment (kyuan-mongodb) [reply src/l1-resources/resource-policy/README.md:6]: I think sticking to the prior language, maybe `use` would be better here.

Comment (EspenAlbert) [reply src/l1-resources/resource-policy/README.md]: UPdated in 121a10d0be1e1183f4d1536886ec2fcf93bd765e.
WDYT @kyuan-mongodb @marcosuma ?

**Review by lantoli — APPROVED** [review]

**Pull request merged**